lib/Bio/KBase/workspace/Client.pm

```diff
@@ -10921,21 +10921,21 @@ auth has a value which is a string
 
 Parameters for the 'list_objects' function.
 
-                At least one of the following filters must be provided. It is strongly
-                recommended that the list is restricted to the workspaces of interest,
-                or the results may be very large:
+                At least one, and no more than 10000, workspaces musts be specified in one of the
```

Comment:
"musts"? Is Gollum writing your docs?

Reply:
Also, what happens if someone puts in 10001 workspaces? Is the 10k limit enforced on the object type, or does the method throw an error?

Reply:
> "musts"? Is Gollum writing your docs?
I actually had this fixed in the spec but forgot to recompile. Anyway, fixed